src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -446,6 +446,10 @@ def visit_If(self, node: ast.If):
             node.orelse = self._process_stmts(node.orelse)
         return node
 
+    def visit_Assert(self, node: ast.Assert):
+        """Assertions are removed in the AssertionChecker later."""
+        return node
+
```

Comment:
Is this really needed? It looks like returning the same node is what the `generic_visit` is also doing. If this is the case, I would remove the node and add the comment to the docstring of the class.

Reply:
If it is removed then it falls back to `visit_Expr`, which triggers an error:
```
        elif call_name not in self.context or not hasattr(self.context[call_name], "_gtscript_"):
>           raise GTScriptSyntaxError("Unknown call", loc=gt_ir.Location.from_ast_node(node))
E           gt4py.frontend.gtscript_frontend.GTScriptSyntaxError: Unknown call

src/gt4py/frontend/gtscript_frontend.py:479: GTScriptSyntaxError
```

Reply:
Ok then